DesignSafe-CI/portal
# Feature: FR Amend/Version

## Overview: ##
Feature branch for FR amends and revisions

## PR Status: ##


## Related Jira tickets: ##

* [DES-2280](https://jira.tacc.utexas.edu/browse/DES-2280)

## Summary of Changes: ##

- There will be group testing sessions for this PR
- Versioning and Amends is now supported for Field Research projects
- Documents for Field Research projects have been removed from the collections manager to their own modal
- Many of the Add/Edit [Project Metadata] modals have been refactored and adjusted to work with versioning and amendments
- Related Works and Associated Projects fields have been added to each publishable type of metadata (projects, missions, experiments etc...); the types that receive dois.
- Conditionally required fields are handled differently and more consistently across the different metadata management modals.

## Testing Steps: ##

- Set up or edit an existing project of any type
- Ensure that editing saves/updates as expected
- Check for any visual bugs/inconsistencies in the projects area
